File: WORDCNT/wordcnt.pl
#!/usr/bin/perl
#
$cases = <STDIN>;

while ( $cases-- )
{
	$line = <STDIN>;
	chomp($line);
	@words = split ( /\s+/ , $line ) ;
        $max_count = 1 ;
        $num = @words ;
        $i = 0  ;
        $prev_len = length $words[0];

        $x = $prev_len;
        while ($i < $num )
	{
                $count = 0 ;
                while ($i < $num )
		{
                        $x = length($words[$i]);
                        if ( $x == $prev_len)
			{
                                $count+=1;
                                $i+=1;
			}
                        else  
			{
                                last;
			}
		}

                $prev_len=$x;
                if ($count > $max_count ) 
		{
                        $max_count = $count ;
		}
	}
        print $max_count,"\n";
	
}

File: WORDCNT/wordcnt.py
#!/usr/bin/python 

cases = int ( raw_input())

while cases : 
	words = raw_input().split(' ')
	max_count = 1 
	num = len( words )
	i = 0  
	prev_len = len(words[0])
	x = prev_len
	while i < num :
		count = 0 
		while i < num   :
			x = len(words[i])
			if x == prev_len :
				count+=1
				i+=1
			else  :
				break
		
		prev_len=x
		if count > max_count : 
			max_count = count 

	print max_count
	cases -=1
			



